docs/home.md

```diff
@@ -1,56 +1,31 @@
 # Overview of TechPass
 
-TechPass is the Identity & Access Management (IAM) and Single Sign-on (SSO) solution for the following [Singapore Government Technology Stack (SGTS) services](#sgts-services-integrated-with-techpass).
+TechPass is the Identity & Access Management (IAM) and Single Sign-on (SSO) solution for the following Singapore Government Technology Stack (SGTS) services.
 
-<a id="sgts-services-integrated-with-techpass">
 
-**SGTS services integrated with TechPass**
-
-</a>
+## SGTS services integrated with TechPass
 
 Following are the list of SGTS services that uses TechPass as their IAM service.
```

Comment:
Cloudscape (https://cloudscape.tech.gov.sg/) is also an SGTS product that's already integrated with TechPass and in production.

Reply:
oh i see , ok

Reply:
Included this to the list.

Reply:
Thanks!